config/migrations/20250212061535-insert-holiday-concepts.graph

```diff
@@ -0,0 +1 @@
+http://mu.semte.ch/graphs/holidays
```

Comment:
Codelist can go into `static`  graph which is currently configured as `http://mu.semte.ch/graphs/redpencil` in the auth config